exercises/solutions/resource_04.py

```diff
@@ -0,0 +1,9 @@
+def open_kiwi_website(page):
+    # 1. Open the kiwi.com website (https://www.kiwi.com/en/)
+    page.goto("https://www.kiwi.com/en/")
+
+    # 2. Accept cookies by clicking the appropriate button
+    page.click("[data-test='CookiesPopup-Accept']")
+
+    # 3. Assert the expected text is displayed
+    assert page.is_visible("text=Book cheap flights other sites simply can’t find.")
```

Comment:
It could, but the initial underlying difference is clearer from the `exercises/tasks/resource_03.py` and `exercises/tasks/resource_04.py`: in nr. 3 the attendee fills just the blanks, in nr. 4 the whole function (even though it's the same code as a result).

Reply:
Discussed locally:

- [ ] Merge both files into one, with the structure of `exercises/tasks/resource_04.py`